### PR TITLE
HTML and language

### PR DESCRIPTION
**Pull request by Artoria2e5**
The first commit should be harmless. It adds a `lang` attribute to pages, and it should really just affect the HTML output's said attribute.

The second commit relies on rms-open-letter/minima#5. The idea is to make the whole thing RTL so that actual markdown syntax can be used more easily. The fallback solution works by creating a div element around all the text when Jekyll's Liquid processor is not present.

If for some reason the minima PR is undesirable, I can just make the div element unconditional.

### CODE REVIEW

File: appendix.fa.md [hunk 1]
```diff
@@ -1,31 +1,35 @@
...
 استالمن سالها علیه استفاده از ضمایر صحیح برای مردم تلاش کرده. این تلاشی رقت انگیز برای پنهان کردن ضدیت با تراجنسهاست. در نسخه اصلی راهبرد ارتباطات محبت آمیز GNU، او می‌گوید: "راههای زیادی وجود دارد که در انگلیسی سوم شخص مفرد را بدون اشاره به جنسیت بیان کنید شما لازم نیست از 'they' استفاده کنید."<a href="https://web.archive.org/web/20210325014959/https://www.gnu.org/philosophy/kind-communication.html">15</a>
 این متن تاکنون بروز شده است، ولی همچنان transphobic است. <a href="https://web.archive.org/web/20210325014959/https://www.gnu.org/philosophy/kind-communication.html">16</a>
 در صفحه اصلی وبسایتش جمله "'They' جمع است - برای اشخاص مفرد، از ضمایر مفرد و خنثی‌جنس استفاده کنید."<a href="https://web.archive.org/web/20210325014851/https://stallman.org/">17</a>
-</p>
+
 <a href="https://rms-open-letter.github.io/index.fa">[برگشت به صفحه اصلی]</a>
+
+{% if false %}
+</div>
+{% endif %}
```
Comment (ehashman) [inline appendix.fa.md:35]:
```suggestion
```

File: index.ar.md [hunk 2]
```diff
@@ -1,33 +1,31 @@
+---
+lang: ar
+---
+
+<!-- Generate a rtl environment under non-Jekyll (e.g. GitHub) -->
+{% if false %}
+<div dir="rtl">
+{% endif %}
```
Comment (ehashman) [inline index.ar.md:8]: Can be removed

File: index.ar.md [hunk 3]
```diff
@@ -1,33 +1,31 @@
...
+لقد قمنا بتفصيل العديد من الحوادث العامة لسلوكيات RMS، البعض منا لديه قصصه الخاصة حول RMS وتفاعلاتنا معه
+، أشياء لم يتم إلتقاطها في ردود الإيمايلات أو في الفيديو، نأمل أن تقرأ ما تمت مشاركته وأن تنظر في الضرر الذي ألحقه في بمجتمعنا والآخرين.
+التفاصيل: https://rms-open-letter.github.io/appendix
+
+للتوقيع يرجى القيام/إرسال digitalautonomy إلى  riseup.net  أو القيام ب
+pull request: https://github.com/rms-open-letter/rms-open-letter.github.io/pulls
+
+{% if false %}
+</div>
+{% endif %}
```
Comment (ehashman) [inline index.ar.md:31]: Can be removed.

File: index.fa.md [hunk 4]
```diff
@@ -1,22 +1,28 @@
+---
+lang: fa
+---
+
+<!-- Generate a rtl environment under non-Jekyll (e.g. GitHub) -->
+{% if false %}
+<div dir="rtl">
+{% endif %}
```
Comment (ehashman) [inline index.fa.md:8]: Can be removed.

File: index.fa.md [hunk 5]
```diff
@@ -1,22 +1,28 @@
...
 <a href="https://rms-open-letter.github.io/appendix.fa">ما برخی وقایع عمومی ناشی از رفتار RMS را اینجا شرح داده‌ایم.</a> بعضی از ما حکایات خودمان درباره RMS و تعاملاتی که با او داشتیم را داریم، وقایعی که در رشته ایمیل ها و یا ویدیو ضبط نشده. ما امیدواریم که شما چیزهایی که در این مورد به اشتراک گذاشته شده بخوانید و آسیبی که او به جامعه ما و دیگران زده است بسنجید.
-</p>
-<p dir="rtl">
+
 برای امضا زدن، لطفا به digitalautonomy at riseup.net ایمیل ارسال کنید یا اینکه <a href="https://github.com/rms-open-letter/rms-open-letter.github.io/pulls">یک Pull request ارائه دهید.</a>
-</p>
+
+{% if false %}
+</div>
+{% endif %}
```
Comment (ehashman) [inline index.fa.md:28]: Can be removed.

**Review by ehashman — CHANGES_REQUESTED** [review]

**Comment by Artoria2e5** [comment]
Sorry for the delay. Working on removing the things now.

**Comment by ehashman** [comment]
Will squash and merge this as a single commit.

**Pull request merged**